controllers/bpfman-agent/ns_application_program.go

```diff
@@ -101,6 +103,17 @@ func (r *NsBpfApplicationReconciler) updateLoadStatus(status bpfmaniov1alpha1.Ap
 // programs on the node via bpfman, and create or update a BpfNsApplicationState
 // object to reflect per node state information.
 func (r *NsBpfApplicationReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	r.Logger = ctrl.Log.WithName("namespace-app")
+	r.interfaces = &sync.Map{}
+	informer := ifaces.NewWatcher(buffersLength)
```

Comment:
It looks like we have a watcher for each reconciler.  Will they both be discovering the same set of interfaces?  If so, would it be possible to have just one for both?

Reply:
moved up one level so it can work for both reconcilers